app/migrations/Version20160519144519.php

```diff
@@ -0,0 +1,42 @@
+<?php
+
+namespace Sylius\Migrations;
+
+use Doctrine\DBAL\Migrations\AbstractMigration;
+use Doctrine\DBAL\Schema\Schema;
+
+/**
+ * Auto-generated Migration: Please modify to your needs!
+ */
+class Version20160519144519 extends AbstractMigration
+{
+    /**
+     * @param Schema $schema
+     */
+    public function up(Schema $schema)
+    {
+        // this up() migration is auto-generated, please modify it to your needs
+        $this->abortIf($this->connection->getDatabasePlatform()->getName() != 'mysql', 'Migration can only be executed safely on \'mysql\'.');
+
+        $this->addSql('DROP INDEX fulltext_search_idx ON sylius_search_index');
+        $this->addSql('CREATE INDEX fulltext_search_idx ON sylius_search_index (item_id)');
+        $this->addSql('DROP INDEX IDX_A29B523F9038C4 ON sylius_product_variant');
+        $this->addSql('ALTER TABLE sylius_product_variant ADD code VARCHAR(255) NOT NULL, DROP sku');
```

Comment:
If you're determined to make this change then I suggest renaming the field to help people upgrading:

```
ALTER TABLE sylius_product_variant CHANGE sku code VARCHAR(255) NUL NULL
```

I still think it's a bad change. 

People will confused Product.code and ProductVariant.code.